test/jestSetup.js

```diff
@@ -1 +0,0 @@
-import 'fast-text-encoding';

```

Comment:
This was used before every test file so I imported it only on places where it's needed and get rid of this setup folder